src/main/kotlin/team/mozu/dsm/domain/item/model/Item.kt

```diff
@@ -0,0 +1,23 @@
+package team.mozu.dsm.domain.item.model
+
+import team.mozu.dsm.domain.annotation.Aggregate
+import java.time.LocalDateTime
+import java.util.UUID
+
+@Aggregate
+data class Item (
+    val id : UUID?,
+    val organId : UUID,
+    val itemName : String,
+    val itemLogo : String,
+    val itemInfo : String,
+    val createdAt : LocalDateTime?,
```

Comment:
생성 시점은 null일 수 없기 때문에 다음과 같이 수정하면 좋을 것 같아요.
```suggestion
    val createdAt : LocalDateTime = LocalDateTime.now(),
```

Reply:
77a2fcc6dd6976bcf0082ff053e672b66733d29f